src/generator/writers/writeFunction.ts

```diff
@@ -599,7 +599,14 @@ export function writeFunction(f: FunctionDescription, ctx: WriterContext) {
                     ctx.context("stdlib");
                 }
 
-                ctx.asm(shuffle, fAst.instructions.join(" "));
+                if (
+                    fAst.instructions.length > 1 ||
```

Comment:
AFAIK, this condition is always false at the moment

Reply:
yes, just added it here in case this gets changed